templates/mainapp/layout.html

```diff
@@ -278,26 +290,58 @@ <h4 class="modal-title" id="booktitle">Basic Modal</h4>
         //for rating a book
         $(function () {
 
-            $(".rateYo").rateYo({
+            $("#rateYo-modal").rateYo({
+                rating: 0,
+                starWidth: "25px",
+                fullStar: true
+            });
+
+        });
+        $(function () {
+
+            $("#rateYo").rateYo({
                 rating: 0,
                 starWidth: "20px",
                 fullStar: true
             });
 
         });
+
+
         // when hover over rate
         function rateStart(obj) {
             var bookid = obj.getAttribute('data-book-id');
             $('#button-text-' + bookid).css('display', 'none');
             $('#rate-' + bookid).css('display', 'block');
         }
         // onclick rate
+        function rateBookModal(obj) {
+            var bookid = obj.getAttribute('data-book-id');
```

Comment:
Could you clear this out? If `bookid` is being captured here, what's the need of using `book__id` in further lines?

Reply:
`bookid` is an empty string because data-book-id doesn't exist for that block of html